source/computer/intel-compiler-note.rst_

```diff
@@ -0,0 +1,7 @@
+.. note::
```

Comment:
Another point that may be included here:

Intel OneAPI is very large ( a few GB), and it may take a lot of time to install it.